test/OpenTelemetry.Instrumentation.Hangfire.Tests/HangfireInstrumentationJobFilterAttributeTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// <copyright file="HangfireInstrumentationJobFilterAttributeTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using Hangfire;
+using Hangfire.MemoryStorage;
+using Moq;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Instrumentation.Hangfire.Tests
+{
+    public class HangfireInstrumentationJobFilterAttributeTests
+    {
+        [Fact]
+        public void Should_Create_Activity()
+        {
+            // Arrange
+            GlobalConfiguration.Configuration
+                .UseSimpleAssemblyNameTypeSerializer()
+                .UseRecommendedSerializerSettings()
+                .UseMemoryStorage();
+
+            var activityProcessor = new Mock<BaseProcessor<Activity>>();
+
+            using var tel = Sdk.CreateTracerProviderBuilder()
+                .AddHangfireInstrumentation()
+                .AddProcessor(activityProcessor.Object)
+                .Build();
```

Comment:
Please use InMemoryExporter in tests.
example: https://github.com/open-telemetry/opentelemetry-dotnet/pull/3122/files

We are going to get rid of any other approaches (Like Moq). from existing tests